docs/build/js-client/6-call-params.md

```diff
@@ -0,0 +1,19 @@
+# Security andd call params
+
+# Call params, tetraplets and security
+
+Each service call is accompanied by additional information specific to Fluence Protocol. Including `initPeerId` - the peer which initiated the particle execution, particle signature and most importantly security tetraplets. All this data is contained inside the last `callParams` argument in every generated function definition. These data is passed to the handler on each function call can be used in the application.
```

Comment:
The last sentence is unclear. It seems a word is lost: "This data is passed to the handler on each function call ??? can be used in the application."

Reply:
Thanks for you comment. I think we have to postpone these section to after the Devner event